packages/config/src/merge-config.ts

```diff
@@ -1,3 +1,9 @@
+/**
+ * This file is a modified version of the `merge-config.ts` file from the panda config package
+ * https://github.com/chakra-ui/panda/blob/main/packages/config/src/merge-config.ts
+ * All it does is remove the methods that only work in a node environment
+ */
+
```

Comment:
I think you meant to put that comment in the `playground/lib/merge-config.ts` file

Reply:
Haha Yes